style.css

```diff
@@ -1,4 +1,41 @@
-.body {
-  padding: 20px;
-  background-color: silver;
+@media (min-width: 320px) {
+  body {
+    font-size: 10px;
+  }
+  a {
+   display: none;
+  }
+  .menu {
+    width: 20px;
+    height: 4px;
+    background-color: blue;
+    margin: 3px 0;
+  }
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 15px;
+  }
+  a {
+   display: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5542841)